### PR TITLE
Add merge option

### PR DESCRIPTION
**Pull request by ETLaurent**
Include `Fix editable override comparator` (https://github.com/punkave/apostrophe-override-options/pull/11/)

### CODE REVIEW

**Comment by ETLaurent** [comment]
Semicolons added!

File: index.js [hunk 1]
```diff
@@ -135,6 +135,22 @@ module.exports = {
...
+          var array = _.get(moduleOptions, sliced);
+          array = _.map(array, function(item) {
+            return _.find(val.$merge, function(replaceItem) {
+              return comparator(replaceItem, item);
+            }) || item;
+          });
+          var additional = val.$merge.filter(function (replaceItem) {
+            return array.every(function (item) {
+              return !comparator(item, replaceItem);
+            })
```
Comment (boutell) [inline index.js]: Please use semicolons consistently at the end of statements here and elsewhere. Otherwise good to go.

**Review by boutell — CHANGES_REQUESTED** [review]
Please use semicolons consistently at the end of statements, otherwise good to go. Thanks.

**Comment by boutell** [comment]
Thanks, merged.

**Pull request merged**